### PR TITLE
Fix invalid attribute

### PR DESCRIPTION
**Pull request by vrajmohan**
Also update .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,10 @@ docs/_build
 regparser.egg-info/
 
 .eregs_index/
+
+# eregs output
+/fr_cache.sqlite
+/diff/
+/layer/
+/notice/
+/regulation/
```
Comment (cmc333333) [inline .gitignore]: I agree with fr_cache.sqlite, but if you're outputting the diff/layer/notice/regulation directories in `.`, you've probably configured something wrong. They probably shouldn't be ignored

File: regparser/notice/xml.py [hunk 2]
```diff
@@ -164,5 +164,5 @@ def notice_xmls_for_url(doc_num, notice_url):
 
 def xmls_for_url(notice_url):
     # @todo: remove the need for this function
-    return [notice_xml._xml
+    return [notice_xml.xml
```
Comment (cmc333333) [inline regparser/notice/xml.py]: Nice catch. I'm betting this means you're using `build_from` over `pipeline`. Let's chat further

File: .gitignore (hunk 1 above)
Comment (vrajmohan) [inline .gitignore]: https://github.com/18F/regulations-parser#using-build_from:
 "Running the command will generate four folders, regulation, notice, layer and possibly diff in the OUTPUT_DIR (current directory by default)."

Comment (cmc333333) [inline .gitignore]: Right, but we don't expect end users to _keep_ these folders in `.`. Using the `pipeline` command, the output must be explicitly defined (as a final parameter) and using `build_from`, we'd expect `OUTPUT_DIR` setting to be modified. If you've output these files into the current directory, you've likely made a mistake, which we don't want to encourage by adding it as a gitignore. Does that make sense?

**Comment by cmc333333** [comment]
Thanks!

**Pull request merged**